kolibri/deployment/default/cache.py

```diff
@@ -12,9 +14,13 @@
 pickle_protocol = OPTIONS["Python"]["PICKLE_PROTOCOL"]
 
 diskcache_location = os.path.join(KOLIBRI_HOME, "process_cache")
-
-diskcache_cache = Cache(diskcache_location, disk_pickle_protocol=pickle_protocol)
-
+try:
+    diskcache_cache = Cache(diskcache_location, disk_pickle_protocol=pickle_protocol)
+except DatabaseError:
+    shutil.rmtree(diskcache_location, ignore_errors=True)
```

Comment:
would it be necessary to call `diskcache_cache.close()` close the connection of the old Cache before or after removing the old cache.db? Thank you!

Reply:
The old db is removed only  it if it's not able to open it because it's corrupted, so it does not need to be closed.

Reply:
thank you for the explanation!